### ISSUE TITLE
Multiple policies with the same name

### ISSUE THREAD

**Issue by yuva29** [contiv/volplugin#469]
When a policy with same name is uploaded twice, the user should be informed about the existing policy otherwise it leaves an impression that the new policy is in use which is not the case!

```
[vagrant@mon0 ~]$ volcli policy upload p1 < /opt/golang/src/github.com/contiv/volplugin/systemtests/testdata/ceph/policy1.json 
[vagrant@mon0 ~]$ volcli policy upload p1 < /opt/golang/src/github.com/contiv/volplugin/systemtests/testdata/ceph/policy2.json 
```

**Comment by erikh** [comment]
Seems good, maybe output the previous and existing policy and confirm? Offer a –y option to skip.

-Erik

**Comment by yuva29** [comment]
So when someone uploads a new policy with the same name, it will affect all the volumes under that policy. I suppose we should not allow this operation itself. WDYT?

**Comment by erikh** [comment]
no, it will only affect the new volumes.

look at hte runtime upload option for how to affect volumes at runtime.

**Comment by yuva29** [comment]
That's right. What I'm trying to say is, its misleading to have volumes that is created from a policy which no more exists. Because, when the user refers the policy, it will be something different than what he used to create volume.

**Comment by erikh** [comment]
This is what the policy archive is for; it can trace back these changes.
